awesome.c

```diff
@@ -543,6 +543,9 @@ main(int argc, char **argv)
     /* We have no clue where the input focus is right now */
     globalconf.focus.need_update = true;
 
+    /* set the default preferred icon size */
+    globalconf.preferred_icon_size = 0;
```

Comment:
> This doesn't really depend on the DPI

Your are right in general, but if the default would be something like 24px, then for a DPI of 180 (instead of 96) the default should be higher.

After all, the default should probably refer to pt and not px?